pandas/_libs/lib.pyx

```diff
@@ -939,6 +939,7 @@ _TYPE_MAP = {
     'float32': 'floating',
     'float64': 'floating',
     'f': 'floating',
+    'complex64': 'complex',
```

Comment:
what system actually hits this? AFAIK numpy doesn't use this on a regular basis

Reply:
Well, it's as easy to use as `float32` on a 64-bit platform. I just added it for completeness.

Reply:
ok see if u can come up with a test then maybe have to force numpy to construct it